spec/requests/internal/configs_spec.rb

```diff
@@ -60,8 +60,33 @@
 
     describe "rate limits" do
       it "updates rate_limit_follow_count_daily" do
-        post "/internal/config", params: { site_config: { rate_limit_follow_count_daily: 3 } }
-        expect(SiteConfig.rate_limit_follow_count_daily).to eq(3)
+        expect do
```

Comment:
Changed the style of this spec because I think the `expect{}.to change(...).from(...).to(...)` form is more in line with what we are testing for, e.g. values do actually change.